### PR TITLE
[ENH] Implemented CDWCE loss

### PR DESCRIPTION
**Pull request by victormvy**
Implemented Class Distance Weighted Cross-Entropy Loss from https://doi.org/10.1007/978-3-031-12053-4_12

### CODE REVIEW

**Review by franberchez — APPROVED** [review]
Thanks for the implementation of this new loss function. It looks great!
